executor/vggish_audio_encoder.py

```diff
@@ -204,7 +202,7 @@ def _get_input_feature(self, batch_docs):
                 blob_shape_list.append(blob.shape[0])
                 mel_list.append(blob)
         elif self._input == 'log_mel':
-            _mel_list = batch_docs.get_attributes('blob')
+            _mel_list = batch_docs.blobs
```

Comment:
`blob` is renamed to `tensor`. https://docs.jina.ai/get-started/migrate/#common-errors-and-solutions
```suggestion
            _mel_list = batch_docs.tensors
```